### PR TITLE
Update mosquitto.mdx for pi

### PR DESCRIPTION
**Pull request by SpudGunMan**
add linux/pi

added raspberry Pi section

same as the Mac just for linux

### CODE REVIEW

File: docs/software/integrations/mqtt/mosquitto.mdx [hunk 1]
```diff
@@ -47,4 +47,54 @@ sidebar_position: 1
...
+   ```
+
+4. In another console, publish a message to that topic:
+
+   ```sh
+   mosquitto_pub -h localhost -q 0 -t test/hello -m 'yo!'
+   ```
+
+5. For Meshtastic to be able to access that server, two settings need to be changed in the
+   `/etc/mosquitto/conf.d/mosquitto.conf` file:
```
Comment (pdxlocations) [inline docs/software/integrations/mqtt/mosquitto.mdx:82]: in my install this was `/etc/mosquitto/mosquitto.conf'

File: docs/software/integrations/mqtt/mosquitto.mdx [hunk 2]
```diff
@@ -47,4 +47,54 @@ sidebar_position: 1
...
+
+   ```shell
+   sudo sh -c "echo 'listener 1883 0.0.0.0\nallow_anonymous true' >> /etc/mosquitto/conf.d/mosquitto.conf"
+   ```
+
+6. Restart the service:
+
+   ```shell
+   sudo service mosquitto restart
```
Comment (pdxlocations) [inline docs/software/integrations/mqtt/mosquitto.mdx:91]: the preferred command is `sudo systemctl restart mosquitto`

File: docs/software/integrations/mqtt/mosquitto.mdx [hunk 3]
```diff
@@ -47,4 +47,54 @@ sidebar_position: 1
...
+   ```
+
+Optionally pull a copy of [pdxlocations/MQTT-Connect-for-Meshtastic](https://github.com/pdxlocations/MQTT-Connect-for-Meshtastic)
```
Comment (pdxlocations) [inline docs/software/integrations/mqtt/mosquitto.mdx:94]: I'm blushing but not sure if this is the place to link a community project?

File: docs/software/integrations/mqtt/mosquitto.mdx [hunk 4]
```diff
@@ -47,4 +47,54 @@ sidebar_position: 1
    brew services restart mosquitto
    ```
 
-7. If you are using the mac firewall, you will need to go into: System Preferences > Security & Privacy > Firewall > Firewall Options and add it.
\ No newline at end of file
+7. If you are using the mac firewall, you will need to go into: System Preferences > Security & Privacy > Firewall > Firewall Options and add it.
+
+
```
Comment (pdxlocations) [inline docs/software/integrations/mqtt/mosquitto.mdx:52]: Perhaps using tabs for various OS's here would be convenient?

File: docs/software/integrations/mqtt/mosquitto.mdx (hunk 1 above)
Comment (SpudGunMan) [reply docs/software/integrations/mqtt/mosquitto.mdx:82]: there was a note in one of the files to use the file I used here, cat the file was that the one?

File: docs/software/integrations/mqtt/mosquitto.mdx (hunk 3 above)
Comment (SpudGunMan) [reply docs/software/integrations/mqtt/mosquitto.mdx:94]: its a very specific client for this purpose and getting it going includes a lot of how-to - your call should I huck it?

File: docs/software/integrations/mqtt/mosquitto.mdx (hunk 1 above)
Comment (pdxlocations) [reply docs/software/integrations/mqtt/mosquitto.mdx:82]: I've recently installed mosquitto on a pi and have no `/etc/mosquitto/conf.d/` folder.  Got a link to that note?

File: docs/software/integrations/mqtt/mosquitto.mdx (hunk 4 above)
Comment (SpudGunMan) [reply docs/software/integrations/mqtt/mosquitto.mdx]: think I added them correctly

**Comment by SpudGunMan** [comment]
image

Tab rendering is funky, not sure what happened

**Review by pdxlocations — APPROVED** [review]

**Pull request merged**